app/services/notification/AWSMailClient.scala

```diff
@@ -0,0 +1,46 @@
+package services.notification
+
+import javax.inject.Inject
+
+import com.amazonaws.handlers.AsyncHandler
+import com.amazonaws.services.simpleemail.AmazonSimpleEmailServiceAsync
+import com.amazonaws.services.simpleemail.model._
+import models.{Instance, Owner}
+import play.api.libs.concurrent.Execution.Implicits._
+import play.api.{Configuration, Logger}
+
+import scala.concurrent.{Future, Promise}
+
+class AWSMailClient @Inject()(amazonMailClient: AmazonSimpleEmailServiceAsync, configuration: Configuration) {
+
+  def send(owner: Owner, instances: Seq[Instance]): Future[String] = {
```

Comment:
Yes you are right!...I thought this was already addressed, but apparently it was neglected!
I will amend!